src/context/ProductContext.jsx

```diff
@@ -5,17 +5,24 @@ import axios from '../config/axios';
 
 export default function ProductContextProvider({ children }) {
   const [items, setItems] = useState([]); // [{}
+  const [category,setCategory] = useState([])
 
   const getItems = () => {
     axios.get('/item').then((response) => {
-      // console.log(response.data);
       setItems(response.data);
     });
   };
 
+  const getCategory = () =>{
+    axios.get(`http://localhost:8000/item/getCategories`).then(response=>{
```

Comment:
```suggestion
    axios.get(`/item/getCategories`).then(response=>{
```

Needs to use standard formatting since we have base url for API already